src/Tribe/Notices.php

```diff
@@ -16,6 +16,17 @@ class Tribe__Notices {
 	 * @return bool
 	 */
 	public static function set_notice( $key, $notice ) {
+
+		/**
+		 * Make notices filterable.
```

Comment:
The addition of an `apply_filters()` call does indeed make the notices filterable but (maybe this is just me) it seems like a slightly odd description to use in the docblock.

Thinking of, say, a 3rd party dev reading this not in the source code but via our technical docs, might a description such as _"Provides an opportunity to alter the text of admin notices"_ provide a little more context?